app/Resources/views/themes/ezplatform/composer/install.html.twig

```diff
@@ -58,13 +58,13 @@
 
         <div class="help">
             <p># Create the database and import clean data</p>
-            {{ Helper.command('php app/console doctrine:database:create') }}
-            {{ Helper.command('php app/console ezplatform:install clean') }}
+            {{ Helper.command('php bin/console doctrine:database:create') }}
+            {{ Helper.command('php bin/console ezplatform:install clean') }}
         </div>
 
         <div class="help">
             <p># You are now going to run eZ Platform with the built-in php server.</p>
-            {{ Helper.command('SYMFONY_ENV=prod php app/console server:run') }}
+            {{ Helper.command('SYMFONY_ENV=prod php bin/console server:run') }}
```

Comment:
this will fail, server:run only works on dev with v2 due to bundle being only loaded on dev (+ more, you know you tried it ;))

So maybe lets skip the `SYMFONY_ENV=prod`, it's most likely there for performance reasons which are fixed now.

Reply:
indeed ;) I am updating the PR right now